src/Writer.php

```diff
@@ -58,7 +58,7 @@ public function exec(): Pair
     /**
      * @template B
      * @param callable $f
-     * @param callable(A): B $f
+     * @psalm-param callable(A):(B) $f
```

Comment:
there is no need to use the parentheses for the output part

Reply:
Ok,I can submit a fix if you want ?

Reply:
no worries, I fixed it, thanks

Reply:
Okay!